Platform/ApollolakeBoardPkg/AcpiTables/Dsdt/Sc.asl

```diff
@@ -145,7 +147,8 @@ scope (\_SB.PCI0) {
       }
     }
     Include("ScPcie.asl")
-    Method(_PRW, 0) { Return(GPRW(0x09, 4)) }  // can wakeup from S4 state
+    Name(_PRF, 0x09)
```

Comment:
Hmm, looks like this ended up a line lower down, it's not consistent with the rest. Can you trivially fix this during merge? If not, I can re-create the PR.

Reply:
So no change at all, right ?    still GPRW(9, 4) ?

Reply:
sorry,  please ignore this comment

Reply:
Nope, good spot, must have missed out on the search/replace here -- should be GPRW(_PRF, 4) like the others for consistency. The aim was to remove the hard-coded GPRW(0x08, 4) from inside ScPcie.asl as RP5 needs 0x03 instead of 0x08.

Reply:
OK